catalog.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<title>Каталог</title>
+</head>
+<body>
+	<header>
+		<nav>
+			<a href="index.html"><img src="img/logo.svg" alt="logo"></a>
+			<ul>
+				<li><a href="catalog.html">Информация</a></li>
+				<li><a href="#">Фото и видео</a></li>
+				<li><a href="#">Карта штата</a></li>
+				<li><a href="#">Гостиницы</a></li>
+			</ul>
+		</nav>
+	</header>
+	<section>
+		<h1>Каталог</h1>
+		<form>
+			<label>Инфраструктура:</label><br>
+			<input id="pond" type="checkbox" checked name="Бассейн">
+			<label for="pond">Бассейн</label><br>
+			<input id="parking" type="checkbox" name="Парковка">
+			<label for="parking">Парковка</label><br>
+			<input id="wifi" type="checkbox" name="Wi-Fi">
+			<label for="wifi">Wi-Fi</label><br>
+			<label>Тип жилья:</label><br>
+			<input id="hotel" type="checkbox" checked name="Гостиница">
+			<label for="hotel">Гостиница</label><br>
+			<input id="motel" type="checkbox" checked name="Мотель">
+			<label for="motel">Мотель</label><br>
+			<input id="apartments" type="checkbox" checked name="Аппартаменты">
+			<label for="apartments">Аппартаменты</label><br>
+			<label>Стоимость в сутки</label>
+			<input type="range" min="0" max="3000" step="1" value="1500">
+			<button type="button">Показать</button>
```

Comment:
```suggestion
			<button type="submit">Показать</button>
```
так как кнопка отвечает за отправку данных из формы.